target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -103,6 +103,13 @@ define Device/ubnt_unifiac-mesh
 endef
 TARGET_DEVICES += ubnt_unifiac-mesh
 
+define Device/ubnt_unifiac-mesh-pro
+  $(Device/ubnt_unifiac)
+  DEVICE_TITLE := Ubiquiti UniFi AC-Mesh Pro
+  SUPPORTED_DEVICES += ubnt_unifiac-mesh-pro
```

Comment:
Shouldn't that be:
SUPPORTED_DEVICES += unifiac-mesh-pro   (to be consistent)
or
SUPPORTED_DEVICES += unifiac-pro   (to enable upgrading)
?

The rest looks fine :-)

Reply:
your commit for ac-mesh support was used as source of inspiration, so i used the consistent method. but you are right, the underscore should be a dash. will fix that asap